sections/mobile-scroll-banners.liquid

```diff
@@ -0,0 +1,97 @@
+<style>
+    .scroll_banner_container{
+        display: none;
+    }
+
+    @media screen and (max-width: 767px) {
+        .scroll_banner_container{
+            display: block;
+            margin-top: 15px;
+            margin-left: 20px;
+        }
+
+        .scroll_banner_container .scroll_card_ele{
+            width: 39.5%;
+            margin-right: 3.5%;
+            border-radius: 10px;
+            counter-increment: scroll_card_ele;
+            overflow: hidden;
+        }
+
+        .scroll_banner_container .scroll_card_container{
+            height: 200px; 
+            width: 100%; 
+            background-size: contain;
+            background-repeat: no-repeat;
+        }
+
+        .scroll_banner_container .scroll_banner_content{
+            position: absolute;
+            bottom: 0px;
+            text-align: center;
+            width: 100%;
+            background-image: linear-gradient(to top, rgba(0, 0, 0, 0.8) 40%, rgba(80, 80, 80, 0.015));
+            color: white;
+            padding-top: 7px;
+            padding-bottom: 7px;
+        }
+
+        .scroll_banner_container .scroll_banner_content h1{
+            font-size: 11.5px;
+        }
+
+    }
+</style>
+
+
+<div class="scroll_banner_container" data-flickity='{ "freeScroll": false, "contain": true, "prevNextButtons": false, "pageDots": false }'>
+    {% for block in section.blocks %}
+
+        <a href="{{block.settings.scroll-ele-link}}" class="scroll_card_ele">
```

Comment:
please add condition Image and Text

Reply:
Conditions have been added , Here is the link for the MR created : https://github.com/AvishekPrasad/amalaearth-test/pull/9